eos-tech-support/eos-fix-ostree-repo

```diff
@@ -151,12 +152,20 @@ def fix_dangling_refs(repo):
             pull_commit(repo, remote, checksum)
 
 
-def mark_commits_partial(repo):
+def get_all_objects(repo):
+    """Get a dictionary of all objects in the repository"""
+    repo_path = os.path.realpath(repo.get_path().get_path())
+    print('Finding all objects in', repo_path)
+
+    return repo.list_objects(OSTree.RepoListObjectsFlags.ALL, None)[1]
```

Comment:
Nitpick: Would maybe be a bit clearer to use tuple matching on the return values:
```
_, all_objects = repo.list_objects(OSTree.RepoListObjectsFlags.ALL, None)
return all_objects
```

Reply:
Sure. I had that first then decided it was noisy :)